Parse/Internal/LocalDataStore/SQLite/PFSQLiteDatabaseResult.m

```diff
@@ -12,11 +12,20 @@
 #import <sqlite3.h>
 
 #import "PFSQLiteStatement.h"
+#import "PFThreadsafety.h"
+
+// PFThreadsafetySafeDispatchSync, but with a return type.
+#define thread_safe_sync(queue, block) ({                          \
```

Comment:
Any chance we can make this a function?
This is not going to be nice (probably), but might feel better than having 2 defines that do the same thing...